ingestion/src/main/java/feast/ingestion/boot/ImportJobModule.java

```diff
@@ -88,7 +98,7 @@ Specs provideSpecs(SpecService.Builder specService) {
 
   @Provides
   @Singleton
-  List<ErrorsStore> provideErrorsStores() {
-    return ErrorsStoreService.getAll();
+  ErrorsStore provideErrorsStores() {
```

Comment:
this should still return a list, as this is about returning the available classes in the class path, which are essential factories for building write transforms. The passed in errorsStoreType is used for selecting the write one from this list.

Reply:
Made changes as suggested